content/docs/prerequisites/_index.md

```diff
@@ -48,49 +48,48 @@ Container Storage Modules (CSM) does not officially support specific operating s
-| ------------------ |---------|-------------------|-----------------|-------------------|----------------|
-| CSI PowerScale     | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0        | ✔ 1.9.0           | ✔ 1.10.0      |
-| CSI PowerScale     | 2.10.1  | ✔ 1.10.1          | ✔ 1.8.1        | ✔ 1.8.1           | ✔ 1.9.1       |
-| CSI PowerScale     | 2.9.1   | ✔ 1.9.1           | ✔ 1.7.1        | ✔ 1.7.0           | ✔ 1.8.1       |
-| CSI PowerFlex      | 2.12.0  | ✔ 1.12.0          | ✔ 1.10.0       | ✔ 1.10.0          | ✔ 1.11.0      |
-| CSI PowerFlex      | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0        | ✔ 1.9.0           | ✔ 1.10.0      |
-| CSI PowerFlex      | 2.10.1  | ✔ 1.10.1          | ✔ 1.8.1        | ✔ 1.8.1           | ✔ 1.9.1       |
-| CSI PowerStore     | 2.11.0  | ❌                | ❌             | ❌                | ✔ 1.10.0      |
-| CSI PowerStore     | 2.10.1  | ❌                | ❌             | ❌                | ✔ 1.9.1       |
-| CSI PowerStore     | 2.9.1   | ❌                | ❌             | ❌                | ✔ 1.8.1       |
-| CSI PowerMax       | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0        | ✔ 1.9.0           | ✔ 1.10.0      |
-| CSI PowerMax       | 2.10.1  | ✔ 1.10.1          | ✔ 1.8.1        | ✔ 1.8.1           | ❌            |
-| CSI PowerMax       | 2.9.1   | ✔ 1.9.1           | ✔ 1.7.1        | ✔ 1.7.0           | ❌            |
-| CSI Unity XT       | 2.12.0  | ❌                | ❌             | ❌                | ❌            |
-| CSI Unity XT       | 2.11.0  | ❌                | ❌             | ❌                | ❌            |
-| CSI Unity XT       | 2.10.1  | ❌                | ❌             | ❌                | ❌            |
+| CSI Driver     | Version | CSM Authorization | CSM Replication | CSM Observability | CSM Resiliency |
+| -------------- | ------- | ----------------- | --------------- | ----------------- | -------------- |
+| CSI PowerScale | 2.11.0  | ✔ 1.11.0, 2.0.0   | ✔ 1.9.0         | ✔ 1.9.0           | ✔ 1.10.0       |
+| CSI PowerScale | 2.10.1  | ✔ 1.10.1          | ✔ 1.8.1         | ✔ 1.8.1           | ✔ 1.9.1        |
+| CSI PowerScale | 2.9.1   | ✔ 1.9.1           | ✔ 1.7.1         | ✔ 1.7.0           | ✔ 1.8.1        |
+| CSI PowerFlex  | 2.12.0  | ✔ 1.12.0, 2.0.0   | ✔ 1.10.0        | ✔ 1.10.0          | ✔ 1.11.0       |
+| CSI PowerFlex  | 2.11.0  | ✔ 1.11.0          | ✔ 1.9.0         | ✔ 1.9.0           | ✔ 1.10.0       |
+| CSI PowerFlex  | 2.10.1  | ✔ 1.10.1          | ✔ 1.8.1         | ✔ 1.8.1           | ✔ 1.9.1        |
+| CSI PowerStore | 2.11.0  | ❌                 | ❌               | ❌                 | ✔ 1.10.0       |
+| CSI PowerStore | 2.10.1  | ❌                 | ❌               | ❌                 | ✔ 1.9.1        |
+| CSI PowerStore | 2.9.1   | ❌                 | ❌               | ❌                 | ✔ 1.8.1        |
+| CSI PowerMax   | 2.11.0  | ✔ 1.11.0, 2.0.0   | ✔ 1.9.0         | ✔ 1.9.0           | ✔ 1.10.0       |
```

Comment:
Same as above, isn't it driver version 2.12.0 and Auth versions 1.12.0 and 2.0.0?

Reply:
fixed